src/Catty/Setup/CatrobatSetup+Bricks.swift

```diff
@@ -89,6 +89,7 @@
             ThinkForBubbleBrick(),
             ThinkBubbleBrick(),
             AskBrick(),
+            SetBackgroundByIndexBrick(),
```

Comment:
Please make sure that this brick is ordered directly below the "Set background" brick.